docs/usage/self-hosted-configuration.md

```diff
@@ -139,6 +139,14 @@ For example:
 }
 ```
 
+## bbUseDevelopmentBranch
```

Comment:
Another open discussion was: do we playn to support more models, so we should use an enum here instead, so we can extend later?

Is there more than one branching model to support in future?

Reply:
The [other branching models ](https://support.atlassian.com/bitbucket-cloud/docs/configure-a-projects-branching-model/) in Bitbucket Cloud are

- production branch - varies what this can be
- feature branch - `feature/*`
- hotfix branch - `hotfix/*`
- release branch - `release/*`
- bugfix branch - `bugfix/*`

I can't think of use-cases where it would be useful to specify these as the target for all Renovate PRs.  What about you?

Reply:
No, I don't think so. In fact they seem to complement the idea of a development branch

Reply:
ok.